### PR TITLE
MGMT-12326: Can't see NHC pause reason in the pop-up window on the status

### PR DESCRIPTION
**Pull request by batzionb**

### CODE REVIEW

**Comment by batzionb** [comment]
Improved pause status popover to show the pause reasons
image
image
image
image

**Comment by batzionb** [comment]
Includes small improvements:
1. Fixed the "Unpause node health check" dialog to show the reasons in bullets same as in popover
2. Fixed all Alerts to be inline
3. Added missing aria labels to TextInput fields to prevent console errors
4. Grammer fixing of node health check conflic with machine health check message

File: src/components/details/NodeHealthCheckStatus.tsx [hunk 1]
```diff
@@ -59,51 +70,114 @@ const getPhaseLabel = (t: TFunction, phase: string): string => {
...
+    <Stack>
+      <StackItem>
+        <Text component={TextVariants.h4} data-test="status-reason">
+          {t("{{count}} pause reason", { count: pauseReasons.length })}
+        </Text>
+      </StackItem>
+      <StackItem>
+        <TextList className="nhc--enhanced-text-list__list">
+          {pauseReasonsToShow.map((reason) => (
+            <EllipsisToolTip content={reason}>
```
Comment (rawagner) [inline src/components/details/NodeHealthCheckStatus.tsx:121]: missing `key`

File: src/components/details/NodeHealthCheckStatus.tsx [hunk 2]
```diff
@@ -59,51 +70,114 @@ const getPhaseLabel = (t: TFunction, phase: string): string => {
...
+              <TextListItem className="nhc--enhanced-text-list__list-item">
```
Comment (rawagner) [inline src/components/details/NodeHealthCheckStatus.tsx:122]:
```suggestion
              <TextListItem className="nhc-enhanced-text-list__list-item">
```

**Review by rawagner — CHANGES_REQUESTED** [review]

**Review by rawagner — APPROVED** [review]

**Pull request merged**